### ISSUE TITLE
OneAway bug

### ISSUE THREAD

**Issue by jrusev** [careercup/CtCI-6th-Edition-Python#11]
[OneAway](https://github.com/careercup/CtCI-6th-Edition-Python/blob/master/Chapter%201/5_One%20Away/OneAway.py#L47) will fail the following tests:

```python
# Check if a string can be converted to another string with a single edit:
data = [
    ('ples', 'pales', True),
    ('pale', 'pas', False) 
]
```

**Comment by bravominski** [comment]
pull request #15 should fix this problem
I also found it yesterday while I was working with these questions.

Not sure how to assign pull requests to ppl. Is it something admin would do?

**Comment by asaph** [comment]
@bravominski Yep. An admin can assign PRs which I just did for PR #15. And it looks like @mmermerkaya has already left you some feedback there.

**Comment by jrusev** [comment]
@bravominski Yes, #15 will fix it if merged as is.

**Comment by mmermerkaya** [comment]
Fixed by #15.
